sdk/openai/openai/samples/v2-beta/javascript/onYourData.js

```diff
@@ -9,6 +9,7 @@
 
 const { AzureOpenAI } = require("openai");
 const { DefaultAzureCredential, getBearerTokenProvider } = require("@azure/identity");
+require("@azure/openai/types");
```

Comment:
Could we not check these in?

Reply:
I'll create an issue for a script to automatically remove this import in js samples file

Reply:
#31358